main/src/ca/uwaterloo/flix/runtime/interpreter/Interpreter.scala

```diff
@@ -346,6 +346,30 @@ object Interpreter {
       if (Solver.entails(v1, v2))
         Value.True else Value.False
 
+    case Expression.FixpointFold(pred, exp1, exp2, exp3, tpe, loc) =>
```

Comment:
I am not sure how well the interpreter works with the fixpoint engine at the moment.  Maybe it works? Maybe it does not?

Just a heads up.

Reply:
It seems to work for solving fixpoints at least. But I'll first focus on the code generation then.